packages/application-generic/src/services/bull-mq/bull-mq.service.ts

```diff
@@ -229,6 +230,38 @@ export class BullMqService {
     });
   }
 
+  public async addBulk(
+    data: [
+      {
+        name: string;
+        data: BullMqJobData;
+        options: BulkJobOptions;
+        groupId?: string;
+      }
```

Comment:
Shouldn't this type be:
```
 {
        name: string;
        data: BullMqJobData;
        options: BulkJobOptions;
        groupId?: string;
      }[]
```
If we are just passing a single element array, I think we should change the declaration of the arguments in the function to like the `add` method in line 214.

Reply:
Not sure i am following here, i would still love it to be an object tho